dbms/tests/queries/0_stateless/01098_msgpack_format.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Do we have a test case for transparent type conversions while reading?
Like - write Array(UInt8) to msgpack, then read as Array(Int64)?